rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -64,6 +64,7 @@ struct BackupDBRequest {
   3: optional i32 limit_mbs = 0,
   # enable appending checksum to sst file name during backup
   4: optional bool share_files_with_checksum = false,
+  5: optional bool backup_with_meta = true,
```

Comment:
Can we rename this to `include_meta` ?

Reply:
Fixed it. change to `with_meta` and add comments to elaborate.